library/src/main/java/com/airbnb/android/airmapview/AirMapView.java

```diff
@@ -295,6 +295,20 @@ public boolean removePolyline(AirMapPolyline polyline) {
     return false;
   }
 
+  public void setGeoJsonLayer(AirMapGeoJsonLayer layer) {
+    if (!isInitialized()) {
```

Comment:
we've been using this pattern for all other methods as well, which is probably not ideal but should work for now